tests/ignite/handlers/test_checkpoint.py

```diff
@@ -320,43 +326,44 @@ def _test(to_save, obj, name):
     _test(to_save, model.state_dict(), "model")
```

Comment:
@divo12 here we can reuse global `model` and create locally `to_save = {"model": model}` and thus remove `_test` and keep its content directly inside `test_checkpoint_with_score_name_only`

Reply:
yep nice idea

Reply:
In fact you reverted a lot of tests, reintroducing `_test` method. The comment was about removing pytest decorator by adapting as @vfdev-5 suggested. Sorry if my comment wasn’t clear.

Reply:
done the changes @vfdev-5

Reply:
i didn't introduce any _test function @sdesrozis